<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureDialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ConfigureDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="configGroupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label0">
        <property name="text">
         <string>identifier:  </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit0"/>
      </item>

      <item row="1" column="0">
       <widget class="QLabel" name="label1">
        <property name="text">
         <string>GD:  </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit1"/>
      </item>

      <item row="2" column="0">
       <widget class="QLabel" name="label2">
        <property name="text">
         <string>sobelovD:  </string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit2"/>
      </item>

      <item row="3" column="0">
       <widget class="QLabel" name="label3">
        <property name="text">
         <string>sobelovW:  </string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit3"/>
      </item>

      <item row="4" column="0">
       <widget class="QLabel" name="label4">
        <property name="text">
         <string>normalD:  </string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit4"/>
      </item>

      <item row="5" column="0">
       <widget class="QLabel" name="label5">
        <property name="text">
         <string>normalW:  </string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEdit5"/>
      </item>

      <item row="6" column="0">
       <widget class="QLabel" name="label6">
        <property name="text">
         <string>itMaxPerIt:  </string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineEdit6"/>
      </item>

      <item row="7" column="0">
       <widget class="QLabel" name="label7">
        <property name="text">
         <string>xtol:  </string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="lineEdit7"/>
      </item>

      <item row="8" column="0">
       <widget class="QLabel" name="label8">
        <property name="text">
         <string>itMax:  </string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="lineEdit8"/>
      </item>

      <item row="9" column="0">
       <widget class="QLabel" name="label9">
        <property name="text">
         <string>mode:  </string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="lineEdit9"/>
      </item>

      <item row="10" column="0">
       <widget class="QLabel" name="label10">
        <property name="text">
         <string>nClosestPoints:  </string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QLineEdit" name="lineEdit10"/>
      </item>

      <item row="11" column="0">
       <widget class="QLabel" name="label11">
        <property name="text">
         <string>treeArgs:  </string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QLineEdit" name="lineEdit11"/>
      </item>

      <item row="12" column="0">
       <widget class="QLabel" name="label12">
        <property name="text">
         <string>fitVerbose:  </string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QLineEdit" name="lineEdit12"/>
      </item>

      <item row="13" column="0">
       <widget class="QLabel" name="label13">
        <property name="text">
         <string>fixedNodes:  </string>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <widget class="QLineEdit" name="lineEdit13"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigureDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigureDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
